.storybook/preview.js

```
@@ -8,3 +7,4 @@ export const parameters = {
 	},
 	layout: 'centered'
 };
+export const tags = ["autodocs"];
```

ESLint

[error] Replace `"autodocs"` with `'autodocs'`

(prettier/prettier)